File: nl.naturalis.nba.api/src/main/java/nl/naturalis/nba/api/model/DistributionLocality.java
package nl.naturalis.nba.api.model;



public class DistributionLocality
{
	private String locality;

	public String getLocality()
	{
		return locality;
	}

	public void setLocality(String locality)
	{
		this.locality = locality;
	}

	
	@Override
	public boolean equals(Object obj)
	{
		if (this == obj) {
			return true;
		}
		if (obj == null || !(obj instanceof DistributionLocality)) {
			return false;
		}
		DistributionLocality other = (DistributionLocality) obj;
		return locality.equals(other.locality);
	}
}

File: nl.naturalis.nba.client.php/model/DistributionLocality.php
<?php

namespace nl\naturalis\nba\client\php\model;

class DistributionLocality extends NBADomainObject {

	private $locality;

	public function getLocality()
	{
		return $this->locality;
	}

	public function setLocality(string $locality)
	{
		$this->locality = $locality;
	}
}
